docs/downloads/automation-library/integrations/jit/review_jit_ignore_accept.cm

```diff
@@ -0,0 +1,23 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+automations:
+  review_jit_ignore_accept:
+    if:
+      - {{ pr.conversations | filter(attr='commenter', term='jit-ci') | map(attr='content') | match(term='#jit_ignore_accept') | some }}
```

Comment:
@BenLloydPearson this line (which i also published), has a bug:
The commenter that writes `#jit_ignore_accept` is not jit-ci, rather is the author of the PR, so in this case we should reject the comments made by jit-ci
in addition, map + match can be combined into a filter
so the line should be as follows:
      - {{ pr.conversations | reject(attr='commenter', term='jit-ci') | filter(attr='content', term='#jit_ignore_accept') | some }}

Reply:
Perfect, this could also explain some wonkiness I experienced while testing this. The most recent commit makes your requested change.